From b29501e75c0187ac7f0c0e4a020c9c4a8797fe5c Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Wed, 24 May 2017 15:38:45 +0800
Subject: [PATCH 1866/5242] MLK-14972-01 doc: dt-bindings: add imx sc temp
 sensor binding doc

commit  5ee91fdc6c88afeb347bb9556600cd1d2afbbf6d from
https://source.codeaurora.org/external/imx/linux-imx.git

Add i.MX8QM/QXP temp sensor binding doc.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/thermal/imx-sc-thermal.txt |   17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/thermal/imx-sc-thermal.txt

diff --git a/Documentation/devicetree/bindings/thermal/imx-sc-thermal.txt b/Documentation/devicetree/bindings/thermal/imx-sc-thermal.txt
new file mode 100644
index 0000000..290e3bc
--- /dev/null
+++ b/Documentation/devicetree/bindings/thermal/imx-sc-thermal.txt
@@ -0,0 +1,17 @@
+* IMX8QM/QXP SoC Temperature Sensor
+
+Required properties:
+- compatible :
+- "nxp,imx8qm-sc-tsens"
+- "nxp,imx8qxp-sc-tsens"
+
+- reg: Address range of the thermal registers
+- #thermal-sensor-cells : Should be 1. See ./thermal.txt for a description.
+- tsens-num: the number of temp sensor on this SOC.
+
+Example:
+tsens: thermal-sensor@ {
+		compatible = "nxp,imx8qm-sc-tsens";
+		tsens-num = <5>;
+		#thermal-sensor-cells = <1>;
+};
-- 
1.7.9.5

